style.css

```diff
@@ -328,7 +328,7 @@ button:hover {
 	-webkit-flex-flow: row wrap;
 	flex-wrap: wrap;
 	-ms-flex-pack: justify;
-	justify-content: space-evenly;
```

Comment:
`space-evenly` is apparently not supported on normal flexed `div` elements in IE and Edge. Surprisingly it still looked fine on IE11 so I just blindly assumed it would also look fine on Edge. Just checked it out randomly and saw it was broken. `space-around` looks very similar so this is a quick and easy fix.